src/libs/ReportUtils.ts

```diff
@@ -2755,6 +2747,32 @@ function getLinkedTransaction(reportAction: OnyxEntry<ReportAction | OptimisticI
     return allTransactions?.[`${ONYXKEYS.COLLECTION.TRANSACTION}${transactionID}`] ?? {};
 }
 
+/**
+ * Check if any of the transactions in the report has required missing fields
+ *
+ */
+function hasMissingSmartscanFields(iouReportID: string): boolean {
+    return TransactionUtils.getAllReportTransactions(iouReportID).some((transaction) => TransactionUtils.hasMissingSmartscanFields(transaction));
```

Comment:
NAB, but I think this can be simplified 
```suggestion
    return TransactionUtils.getAllReportTransactions(iouReportID).some(TransactionUtils.hasMissingSmartscanFields);
```